src/integrations/third-party/web-stories.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Conditionals\Web_Stories_Conditional;
+use Yoast\WP\SEO\Integrations\Front_End_Integration;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * Web Stories integration
+ */
+class Web_Stories implements Integration_Interface {
+
+	/**
+	 * The front end integration.
+	 *
+	 * @var Front_End_Integration
+	 */
+	protected $front_end;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_conditionals() {
+		return [ Front_End_Conditional::class, Web_Stories_Conditional::class ];
+	}
+
+	/**
+	 * Constructs the Web Stories integration
+	 *
+	 * @param Front_End_Integration $front_end The front end integration.
+	 */
+	public function __construct( Front_End_Integration $front_end ) {
+		$this->front_end = $front_end;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function register_hooks() {
+		\add_action( 'web_stories_story_head', [ $this, 'remove_web_stories_meta_output' ], 0 );
```

Comment:
Are these slashes at the start required?

Reply:
Technically not necessary, but there's a slight change in meaning (and probably performance)  between `\add_action` and `add_action`.

Also, it's how the rest of the plugin is written, so  just following the coding standards here.